### ISSUE TITLE
Empty audios

### ISSUE THREAD

**Issue by ignacio-ferreira-dev** [tugstugi/pytorch-dc-tts#20]
Hi, I am doing custom training for spanish with a custom dataset I created using youtube-tts-data-generator from youtube.
I changed the vocabulary and trained both the text2mel and ssrn models for 20k and 5k iterations.
The loss function for the text2mel model doesnt seem to be improving anymore.
When I synthesize some phrases all are blank and audios have no data in them.
What could I be doing wrong? 
I just need to generate audio using one voice in this case.
I used the prepro.py from dc-tts repo for generating mel and mags as the part that does the preprocessing on this repo is on todo.
Could it be a problem with that?
Thanks!

**Comment by tugstugi** [comment]
You need to provide more information. Use only preprocessing scripts from this repo. Look also into the log folder. During training, some spectrogram images should be created.

**Comment by ignacio-ferreira-dev** [comment]
Ok, I tried with the repo preprocessing but got the same results.
Here is what I got so far: 
![audio](https://user-images.githubusercontent.com/73884196/108792460-16842000-7560-11eb-87cf-dda50af300ba.png)
![train](https://user-images.githubusercontent.com/73884196/108792463-17b54d00-7560-11eb-96cb-b0ca019f37d8.png)
All audio I generate gives me something like this with an empty audio:
1-att.png
![1-att](https://user-images.githubusercontent.com/73884196/108792585-606d0600-7560-11eb-841c-0ed0d3df056e.png)
1-mag.png
![1-mag](https://user-images.githubusercontent.com/73884196/108792588-606d0600-7560-11eb-849d-d8f849b046f2.png)
1-mel.png
![1-mel](https://user-images.githubusercontent.com/73884196/108792589-61059c80-7560-11eb-883b-76d96a1f17f2.png)

And show a warning:
/home/user/Downloads/Code/pytorch-dc-tts/utils.py:73: UserWarning: samples/1-mel.png is a low contrast image
/home/user/Downloads/Code/pytorch-dc-tts/utils.py:73: UserWarning: samples/1-mag.png is a low contrast image

**Comment by ignacio-ferreira-dev** [comment]
As an update, I checked the dataset generated and there are 3 elements per row seperated by |, the second one had the data I needed with all the accents. I modified the code to use this instead of the third element. Also the text normalization applied removed all accents from the input so I removed that too. Now after 75k and 16k I am getting some audios. I am not sure why but still some of the outputs are still empty and the audios generated are not really clear yet.
Could it be that more data is needed or more training?

**Comment by ignacio-ferreira-dev** [comment]
Ok, good to know. I started training with 36 hours of data the model from scratch. I will then finetune to the wanted voice. In case it doesn't work will try modifying the already trained models but for now I will try with this approach.
I will see how it goes!

**Comment by ignacio-ferreira-dev** [comment]
I was able to generate audio in spanish by using more data!
I used 37.5 hours of audio and started working in 182k iterations of the text2mel model.
I had to retrain a lot also the ssrn model but they ended up working.
Thanks tugstugi!
